packages/nx/src/utils/sync-generators.ts

```diff
@@ -272,6 +278,10 @@ export function getSyncGeneratorSuccessResultsMessageLines(
         result.outOfSyncMessage ?? `Some files are out of sync.`
       }`
     );
+
+    if (logOutOfSyncDetails && result.outOfSyncDetails?.length) {
+      messageLines.push(...result.outOfSyncDetails);
+    }
```

Comment:
This is sort of unrelated but can we also add some instruction to run `nx sync` to actually correct these check failures?

1. I think it's good to lead users.
2. I think AI will pick it up and make this easier to fix for AI :shrug: